### DISCUSSION TITLE
Trouble importing self-generated 'stablelm-interface' Python package

### DISCUSSION THREAD

**Discussion by kvothesson** [community/community#53659] [Packages]
### Select Topic Area

Question

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

I have generated a Python package called 'stablelm-interface' and I am having trouble importing it. 

```
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-9-70e8cb42da53>](https://localhost:8080/#) in <cell line: 1>()
----> 1 import stablelm_interface

ModuleNotFoundError: No module named 'stablelm_interface'
```

I have tried all of the typical troubleshooting steps, such as uninstalling and reinstalling the package, verifying that it is installed in the correct environment, and checking my import statement for typos. However, I am still unable to import the package. Could someone please advise me on what else I can do to troubleshoot this issue? Thank you

![image](https://user-images.githubusercontent.com/46162626/234199464-fbf65473-4a40-45f1-8ade-9595a6783c4d.png)
![image](https://user-images.githubusercontent.com/46162626/234199524-c6028b67-afbc-4eb7-9a50-fcfa3df79d72.png)
![image](https://user-images.githubusercontent.com/46162626/234199586-1bb3827a-b194-47f2-854f-dfce8f0bf520.png)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by airtower-luna** [comment]
Exactly *how* did you install the package?

Also I notice that the module name inside your `src/` directory is `stablelm_interface_kvothesson`, so (assuming the package is installed correctly) that's the name you'd need to include.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ezequielsobrino ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
